mail/api/account.py

```diff
@@ -60,53 +54,26 @@ def get_account_request(request_key: str) -> dict:
 def create_account(request_key: str, first_name: str, last_name: str, password: str) -> None:
 	"""Create a new user account"""
 
-	account_request, email, tenant, role = frappe.db.get_value(
-		"Mail Account Request", {"request_key": request_key}, ["name", "email", "tenant", "role"]
+	account_request, email, tenant, role, account = frappe.db.get_value(
+		"Mail Account Request", {"request_key": request_key}, ["name", "email", "tenant", "role", "account"]
 	)
```

Comment:
For invalid request key:
 
![image](https://github.com/user-attachments/assets/b97a53fd-cf7f-48c6-9b75-79ff98bece2f)